src/coreclr/nativeaot/Runtime/yieldprocessornormalized.cpp

```diff
@@ -14,105 +14,6 @@
 #include "slist.h"
 #include "volatile.h"
 #include "yieldprocessornormalized.h"
+#include "../../vm/synch.h"
```

Comment:
What do you need from the sync.h header file? It would be best to avoid it pulling into naot.

Reply:
Yea actually at some point I thought I needed it but I think I can be removed